.github/workflows/dummy_commit.yml

```diff
@@ -0,0 +1,24 @@
+name: Dummy commit
+
+on:
+  schedule:
+    # Triggers the workflow every 15 days at 5:00 UTC:
+    - cron: "0 5 15 * *"
+  workflow_dispatch:
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    env:
+      CI_COMMIT_MESSAGE: Continuous Integration Build trigger
+      CI_COMMIT_AUTHOR: Continuous Integration
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.ROBOT_PAT_TRIGGER_E2E_WORKFLOWS }}
```

Comment:
I don't think we need a PAT here. If default permissions of the default `GITHUB_TOKEN` aren't enough, you can set `contents` to `write` and it should be enough.
https://docs.github.com/en/actions/security-guides/automatic-token-authentication

Reply:
The point of adding this PAT was to allow the commit to trigger workflow ([which isn't the case when you checkout with the default one](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow)) but maybe we don't need this job to trigger workflow. I don't know if it gonna work as there is no documentation on that :x